tests/keras_tests/function_tests/test_activation_weights_composition_substitution.py

```diff
@@ -231,12 +231,12 @@ def test_all_weights_layers_composition(self):
         self.assertTrue(len(sorted_v_nodes[1].candidates_quantization_cfg) == 9)
         # Conv2-Activation node
         self.assertTrue(isinstance(sorted_v_nodes[2], VirtualSplitActivationNode))
-        self.assertTrue(len(sorted_v_nodes[2].candidates_quantization_cfg) == 1)
+        self.assertTrue(len(sorted_v_nodes[2].candidates_quantization_cfg) == 3) # since we do not run fusion, there are three possible activations for the conv
```

Comment:
Doesn't this ruin the test? in there are 3 activations then no composition would happen and the test isn't testing anything.
Why not run fusing?

Reply:
I thought you wanted to remove the fusing from this test, no?